scripts/!mods_preload/msu_aa.nut

```diff
@@ -1,21 +1,12 @@
-local gt = this.getroottable();
+::MSU <- {};
+::MSU.Version <- "1.0.0-alpha";
+::MSU.ID <- "mod_MSU";
```

Comment:
We should discuss what should be our id. Should it be `mod_MSU` or simply `msu`?

Reply:
I suppose "msu" would be fine.